creator-node/src/monitors/monitors.js

```diff
@@ -0,0 +1,100 @@
+const { getDatabaseSize, getDatabaseConnections } = require('./database')
+const { getTotalMemory, getUsedMemory } = require('./memory')
+const {
+  getFilesystemSize,
+  getFilesystemUsed,
+  getMaxFileDescriptors,
+  getAllocatedFileDescriptors
+} = require('./filesystem')
+const {
+  getReceivedBytesPerSec,
+  getTransferredBytesPerSec
+} = require('./network')
+const {
+  getRedisNumKeys,
+  getRedisUsedMemory,
+  getRedisTotalMemory
+} = require('./redis')
+
+// Prefix used to key each monitored value in redis
+const MONITORING_REDIS_PREFIX = 'monitoring'
+
+/**
+ * List of all monitors to run, containing:
+ *  @param {string} name A unique name for the metric (for caching in redis)
+ *  @param {function} func The actual work to compute a value. The return value is what is cached.
+ *  @param {number?} ttl TTL in seconds for how long a cached value is good for.
+ *    Since the job runs on a cron, the min TTL a metric can be refreshed is 60s.
+ *    If a TTL isn't provided, the metric is refreshed every 60s.
```

Comment:
jw, how did you decide the ttl per criteria?

Reply:
totally came out of my head. lmk if you have more thoughts. figured we'd probably tweak, but something like disk size is likely to fill up instantly. more expensive monitors to calculate could be done at lower intervals too. 🤷